packages/palette/src/elements/Radio/Radio.tsx

```diff
@@ -94,9 +95,12 @@ export const Radio: React.FC<RadioProps> = ({
         </Flex>
 
         {isText(description) ? (
-          <Text variant="xs" color="black60">
-            {description}
-          </Text>
+          <Flex>
+            <Spacer ml="30px" />
```

Comment:
I don't see this in the design? Can you highlight in the screenshot where this shows up? Nor do I think we should be using a value of `30px` here.

Reply:
Yeah, I'd absolutely avoid that. You could use a simple CSS grid here.

Reply:
Worth noting that 30px isn't in the [spacing scale of the design system](https://palette.artsy.net/atoms/spacing/#all-spacing-values) either, so that should be a bit of a flag